src/main/views/search-results.njk

```diff
@@ -0,0 +1,38 @@
+{% extends "template.njk" %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% block pageTitle %}
+  {{ "Search Results" | safe }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    text: "Back",
+    href: "/search-option"
+  }) }}
+{% endblock %}
+
+{% block content %}
+  <h1 class="govuk-heading-l">Courts or tribunals in {{ searchInput }}</h1>
+  <p class="govuk-body">We have found {{ searchResults.length }} courts or tribunals matching your search for
+    '{{ searchInput }}'</p>
+
+  <table class="govuk-table govuk-!-width-two-thirds">
+    <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th scope="col" class="govuk-table__header">Court or tribunal</th>
+      <th scope="col" class="govuk-table__header--numeric">Number of hearings</th>
```

Comment:
I'm getting an issue where the header line / border-bottom of the numeric column doesn't appear.

Looks like govuk-table__header--numeric only does the right align - so also need the govuk-table__header class as well